features/collection.feature

```diff
@@ -6,6 +6,7 @@ Feature: Collections support
   @createSchema
   Scenario: Retrieve an empty collection
     When I send a "GET" request to "/dummies"
+    Then print last response
```

Comment:
Don't print response out of development. Moreover, better than adding such scenario you should be able to debug failing responses by launching Behat with `-vvv`.